stock_request_seller_check/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to create stock request for only products that have seller.
```

Comment:
```suggestion
This module limites the user of stock requests only to seller products, since there is no business scenario where stock requests should be created for non-seller products.
```